clients/java/src/main/java/io/camunda/zeebe/client/api/worker/JobClient.java

```diff
@@ -149,4 +150,25 @@ public interface JobClient {
    * @return a builder for the command
    */
   ThrowErrorCommandStep1 newThrowErrorCommand(ActivatedJob job);
+
+  /**
+   * Command to activate multiple jobs of a given type.
+   *
+   * <pre>
+   * jobClient
+   *  .newActivateJobsCommand()
+   *  .jobType("payment")
+   *  .maxJobsToActivate(10)
+   *  .workerName("paymentWorker")
+   *  .timeout(Duration.ofMinutes(10))
+   *  .send();
+   * </pre>
+   *
+   * <p>The command will try to use {@code maxJobsToActivate} for given {@code jobType}. If less
+   * then the requested {@code maxJobsToActivate} jobs of the {@code jobType} are available for
+   * activation the returned list will have fewer elements.
+   *
+   * @return a builder for the command
+   */
+  ActivateJobsCommandStep1 newActivateJobsCommand();
```

Comment:
Thank you 🙂
But I think that this will break backward compatibility from the user's perspective.

Someone could use the method and if we remove it we should up the major version. WDYT?

Reply:
Yes, you are right, missed it, thanks :)